content/05.discussion.md

```diff
@@ -1,18 +1,22 @@
 ## Discussion
-PRS holds much promise for development of new precision medicine approaches by identifying high risk individuals who may benefit from prioritized interventions.
-Nevetheless, one of its current limitation is the model simplicity [@doi:10.1038/s41576-018-0018-x].
-As a first step towards addressing this issue and increasing comprehensiveness of risk profiling models, in this study, we developed a new model that enables automatic genotype encodings and takes into account gene-gene interactions.
+
+We introduce a Multilocus Risk Score (MRS) method to improve the performance of the standard PRS in disease risk stratification of patient populations. 
+While PRS holds much promise for development of new precision medicine approaches by identifying high risk individuals who may benefit from prioritized interventions, one of its current limitations is the model simplicity [@doi:10.1038/s41576-018-0018-x].
+As a first step towards addressing this issue and increasing comprehensiveness of risk profiling models, in this study, we developed a new applied MRS method from the MB-MDR framework that enables automatic genotype encodings and takes into account multiple models for detecting gene-gene (GxG) interactions. 
 Utilizing the efficient implementation of MB-MDR, MRS automatically infers the genotype encodings and simultaneously computes the risk matrices of pairs of variants.
+Through comparing method performance on unbiased collections of simulated data, we demonstrate the robust polygenic risk profiling ability of MRS and suggest the importance of flexible, precise methods in better capturing epistasis behind individual patient risk.
 
+#### The Advantages of MRS
 We showed that the MRS method outperformed standard PRS in many of the simulated datasets, highlighting the importance of genotype encodings and consideration of epistasis.
 We further examined the association between this improvement and the amount of two-way epistatic effect induced in the binary phenotypic outcome.
 Appropriate phenotype encodings are important for improving the accuracy when there is a large amount of main effects of the variants on the phenotypic outcome.
 Meanwhile, inclusion of epistatic terms significantly increases the accuracy from PRS, especially when two-way interactions are present in the data.
-Although we only considered up to two-way interactions, it is straightforward to incorporate higher order interactions (e.g. three-way, four-way) into MRS.
-However, preliminary analyses on the simulated datasets did not show much improvement from the current MRS (results not shown).
+Although we only considered up to two-way GxG interactions, it is straightforward to incorporate higher order interactions (e.g. three-way, four-way) into MRS.
```

Comment:
Have we used GxG before? I think I have been writing gene-gene. I'm fine with either. We just need to be consistent.

Reply:
I see you started using GxG in the Introduction, so this works! I'll adjust some wordings in Methods later.